ldb/templates/ldb/person_detail.html

```diff
@@ -103,17 +103,29 @@ <h2 class="page-header">
         {% if person.ldap_username or person.netid or person.facebook_id or person.linkedin_id %}
           <fieldset>
             <legend>Accounts</legend>
-            {% if person.ldap_username and user|has_group:'userman2' %}
-              {% stcontrol_format person 'ldap_username' '<a href="https://userman2.ch.tudelft.nl/users/{0}">{0}</a>' %}
-            {% else %}
-              {% stcontrol person 'ldap_username' %}
+            {% if person.ldap_username %}
+              <div class="form-group">
+                <label class="col-sm-4 control-label">LDAP-gebruikersnaam</label>
+                <div class="col-sm-8">
+                  <p class="form-control-static">
+                    {% if user|has_group:'userman2' %}
+                      <a href="https://userman2.ch.tudelft.nl/users/{{ person.ldap_username }}">{{ person.ldap_username }}</a>
+                    {% else %}
+                      {{ person.ldap_username }}
+                    {% endif %}
+                    {% if person.email_forward %}
+                      <br/><small>E-mail wordt doorgestuurd naar {{ person.email }}</small>
```

Comment:
Hmmmm... is this the most logical place to show the mail forward?

I think this should be in a `{% trans %}` template tag, so we are prepared for when we should switch to English

Reply:
Updated :slightly_smiling_face: